.github/workflows/docker.yml

```diff
@@ -9,8 +9,9 @@ on:
   push:
     branches:
       - main
-      - release/*
       - cloud/*
+      - release/*
+      - feature/*
```

Comment:
Added the `feature/` prefix for branches we want to publish that are not a release or for cloud.